src/components/WeatherWidgetForecastCard.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="forecast-card">
+    <h4 class="forecast-card__day">{{ props.day }}</h4>
+    <div class="forecast-card__image">
+      <img :src="props.weatherImage" :alt="`Weather Image for ${props.day}`" />
```

Comment:
Let's try to put props on separate lines if there are 2 or more of them. Like so:

```
<img
  prop1="value1"
  prop2="value2"
/>
```